CONTRIBUTING.md

```diff
@@ -2,31 +2,109 @@
 
 First, thanks for taking the time to contribute to the project! Your support is highly appreciated - kudos!
 
-The following is a set of guidelines for contributing to KubeCF and its modules. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+The following is a set of guidelines for contributing to KubeCF and its modules. These are mostly guidelines, not rules.
```

Comment:
Please, also update https://github.com/cloudfoundry-incubator/kubecf-docs/blob/master/content/en/docs/Contribution%20guidelines/_index.md as you see fit